gradle/libs.versions.toml

```diff
@@ -1,18 +1,18 @@
 [versions]
 animalSniffer = "1.6.0"
-arrowGradleConfig = "0.11.0-alpha.1"
-assertj = "3.23.1"
+arrowGradleConfig = "0.12.0-rc.3"
+assertj = "3.24.2"
 coroutines = "1.7.1"
-classgraph = "4.8.151"
+classgraph = "4.8.160"
 dokka = "1.8.10"
 jUnit = "4.13.2"
-jUnitJupiter = "5.8.2"
-jUnitVintage = "5.8.2"
-kotest = "5.5.4"
-kotestGradle = "5.5.4"
+jUnitJupiter = "5.9.2"
+jUnitVintage = "5.9.2"
+kotest = "5.5.5"
+kotestGradle = "5.5.5"
```

Comment:
I think this version had issues on native (missing klib)

Reply:
Indeed. Let's try with the 5.6.x series.